jsonschema-validation.xml

```diff
@@ -454,36 +450,17 @@
                     </t>
                 </section>
 
-                <section title="dependencies">
+                <section title="requiredDependencies">
                     <t>
-                        <cref>
-                            This keyword may be split into two, with the variation that uses
-                            an array of property names rather than a subschema getting a new
-                            name.  The dual behavior is confusing and relatively difficult to
-                            implement.  In the previous draft, we proposed dropping the keyword
-                            altogether, or dropping one of its forms, but we received feedback
-                            in support of keeping it.  See issues #442 and #528 at
-                            &lt;https://github.com/json-schema-org/json-schema-spec/issues&gt;
-                            for further discussion.  Further feedback is encouraged.
-                        </cref>
+                        This keyword specifies property requirements that are conditional
+                        based on the presence of other properties.
```

Comment:
I'm not sure this sentence is useful considering the one that follows (in fact, I cannot easily read it (but maybe it's because I'm not native English)). Also, most keyword's description start out by "The value of this keyword MUST be ..." so this one would be (is already) an exception with this sentence. Not a big deal, though :)

Reply:
Good points, let me see if I can rework it to be more consistent and clear.